bridges/RaceDepartmentBridge.php

```diff
@@ -6,17 +6,21 @@ class RaceDepartmentBridge extends FeedExpander {
 	const MAINTAINER = 't0stiman';
 
 	public function collectData() {
-		$this->collectExpandableDatas('https://www.racedepartment.com/news/archive.rss', 10);
+		$this->collectExpandableDatas('https://www.racedepartment.com/ams/index.rss', 10);
 	}
 
 	protected function parseItem($feedItem) {
-		$item = parent::parseItem($feedItem);
+		$item = parent::parseRSS_2_0_Item($feedItem);
```

Comment:
Why not parseItem, btw?

Reply:
i vaguely remember there was a reason for this, but if i change it back now it doesn't seem to make any difference :thinking:.
i'll leave it like this for now since it works but I'll switch it back on the next PR (if there will be one).